activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -4,6 +4,22 @@
 
 module I18n
   class Railtie < Rails::Railtie
+    module Helpers
+      def self.extensions_for(glob)
+        $1.split(',') if glob =~ /\{([\S]+)\}/
```

Comment:
```
Demo::Application.config.paths["config/locales"].glob
=> "*.{rb,yml}"
```

We need to pass these extensions in array format (`["rb", "yml"]` to `ActiveSupport::FileUpdateChecker` in order to watch the files. That's what `extensions_for` does.